packages/customers/src/Nav.tsx

```diff
@@ -40,7 +40,7 @@ const Nav: FC = () => {
       <NavLink
         end={false}
         to={AppRoute.Customers}
-        icon={<Customers fontSize="small" />}
+        icon={<Customers color="primary" />}
```

Comment:
this is what I was trying to avoid, by defaulting to the primary colour. 
do we have to specify a colour now, every time we use an icon? I would expect the icons to default to the primary colour

Reply:
on reflection.. probably ok to go with the mui default which doesn't specify colour. 
original thinking was that we almost always want the icons to be the primary colour, and so to prevent errors and make them easier to use we should just default to the primary